Classifier and Summarizer/WebEntityDetection.py

```diff
@@ -0,0 +1,120 @@
+import requests
```

Comment:
We should also try to PEP 8 guidelines for method/variable names (https://www.python.org/dev/peps/pep-0008/#naming-conventions), docstring (https://www.python.org/dev/peps/pep-0008/#documentation-strings), whitespaces (https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).